QuantumToolbox.jl/time_evolution/fluorescence.qmd

```diff
@@ -0,0 +1,230 @@
+---
+title: "Resonance fluorescence"
+author: Li-Xun Cai
+date: 2025-01-27 # last update (keep this comment as a reminder)
+
+engine: julia
+---
+
+Inspirations taken from [this QuTiP tutorial](https://nbviewer.org/urls/qutip.org/qutip-tutorials/tutorials-v5/lectures/Lecture-13-Resonance-flourescence.ipynb) by J. R. Johansson.
+
+In this tutorial, we demonstrate the following functionality:  
+
+- `QuantumToolbox.liouvillian`,  
+- `QuantumToolbox.mesolve`,  
+- `QuantumToolbox.correlation_2op_1t`  
+- `QuantumToolbox.spectrum_correlation_fft` 
+
+with the example of resonance fluorescence in the weak field limit. Note that we set $\hbar = 1$ throughout.  
+
+## Introduction
+
+Resonance fluorescence is the radiative behaviour of a two-level atom driven by a resonating light mode in coherent state (usually a monochromic laser beam). The Hamiltonian $H$ is given by  
+
+- $H_\text{a} = \frac{\omega_a}{2} \sigma_z$
+- $H_\text{drv} = \Omega \cos(\omega_L t) (\sigma^{+} + \sigma^{-})$
```

Comment:
I think the hats are missing above the operators. This should be also done in the other equations.